addons/hr_attendance/tests/test_hr_attendance_report.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import time
+from openerp.tests import common
+from openerp.tools import test_reports
+
+
+class TestHrAttendanceReport(common.TransactionCase):
+
+    def setUp(self):
+        super(TestHrAttendanceReport, self).setUp()
+
+    def test_hr_attendance_report(self):
+        # Print HR Attendance Error Report through the wizard
+        ctx = {'model': 'hr.employee', 'active_ids': [self.ref('hr.employee_fp')]}
+        data_dict = {'init_date': time.strftime('%Y-01-01')}
+        test_reports.try_report_action(self.cr, self.uid, 'action_hr_attendance_error', wiz_data=data_dict, context=ctx, our_module='hr_attendance')
```

Comment:
Is there any need of this test case,
check here:  https://github.com/odoo/odoo/blob/master/addons/report/tests/test_reports.py
It is already testing all qweb reports